### PR TITLE
Adding deep dive video for Docusaurus

### PR DESCRIPTION
**Pull request by dmitryvinn**
## Motivation

Following recent update with ELI5 video, I wanted to see if you would be opened to add a longer version of the video to the home page that we presented at the Algolia meetup

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?
Yes

## Test Plan

image

## Related PRs

#6659

### CODE REVIEW

**Comment by Josh-Cena** [comment]
@lex111 @yangshun @slorber Any better design you can think of? We probably want something nicer in the spirit of #6386. The jest website may be a good reference with two videos

**Comment by lex111** [comment]
Two videos side by side in the viewport seems too overwhelming to me. I think the deep dive video should be in the index page of docs rather than on the landing page. Ideally, the video content place in the middle of the page, so that a visitor can first look at the features. WDYT?

**Comment by Josh-Cena** [comment]
Same idea here. If it's more informative than promotional, we probably don't want to put it on the front page. However, Jest does have a 1:38:31 talk on their landing page and it doesn't look terrible, considering how they've set up their layout.

(Fun thing is, the entire iframe on the Jest website is invisible on mobile. Probably a bug)

**Comment by slorber** [comment]
Also not sure it's a good video for efficient marketing on the homepage


> I don't think the deep dive video should be on the homepage. Maybe on the docs intro page or [Advanced Guides](https://docusaurus.io/docs/advanced)?

I don't think it's deep enough to be on the advanced guides, but on the intro page that looks like a good fit.

What about adding this under or above our fast track section?
https://docusaurus.io/docs#fast-track

Maybe add a new section like "Video intro"?

**Comment by SimenB** [comment]
> @SimenB Do you know if that's intentional?

No idea 😀

**Comment by Josh-Cena** [comment]
I guess it's to ensure a reasonable scroll distance for mobile? That makes sense, because otherwise you would get a super long scroll as on the Docusaurus site.

**Comment by SimenB** [comment]
If you land on some better layout here I'm sure we'd be happy to copy that approach for Jest's site as well

**Comment by Josh-Cena** [comment]
Good design comes out of Orta, so I'd rather follow what the Jest website does😆

**Comment by Josh-Cena** [comment]
So the consensus seems to be to move this to the intro docs, probably the "fast track" section, cool. Does it have to be an iframe? Or is a plain link fine?

**Comment by SimenB** [comment]
>Not sure it's linked to from anywhere since I only discovered it through source code

Ergh, it should be in the sidebar

**Comment by dmitryvinn** [comment]
I see - I can definitely move it there, thank you! We can use a link like I did with Detectron2 that only has README.md and no site, but I personally like when I can play a video on the same page. However, you are the project maintainers, I will listen to any decision you will make

[Explain Like I’m 5: Detectron2](https://www.youtube.com/watch?v=1oq1Ye7dFqc)

**Comment by slorber** [comment]
Personally I'd add it there:

image

I'd rather order those things by how much time they need.
It's longer to watch the video so it probably makes sense to make this video less visible than the fast-track tutorial. 

I'd add it under a new "Video Intro" section, also explaining what this video is about and why it's worth watching in 1 or 2 lines. + embedded video player

Does anyone disagree?

**Comment by dmitryvinn** [comment]
Thank you all - how does this look?
image

**Comment by slorber** [comment]
Thanks

Going to merge and move on for now as it's good enough

I'm not 100% convinced this section heading is the best though, maybe we should make it clearer this section is a video, like "Video Intro" or something?
